src/test/java/com/speaker/utils/FriendGenerator.java

```diff
@@ -0,0 +1,13 @@
+package com.speaker.utils;
+
+import com.speaker.entities.Friend;
+
+public class FriendGenerator {
+    public static Friend generateFriend(int accountId, int friendId){
```

Comment:
no need to extract this little method from the test

Reply:
Deleted class. Used a builder